app/views/shared/_left_menu_bar.html.erb

```diff
@@ -81,9 +81,6 @@
           <li><%= link_to t('left_menu_bar.support_links.tutorials'),
                             Constants::TUTORIALS_URL,
                             target: "_blank" %></li>
-          <li><%= link_to t('left_menu_bar.support_links.webinars'),
-                            Constants::WEBINARS_URL,
-                            target: "_blank" %></li>
```

Comment:
Should we also remove constant if it's not needed anymore?

Reply:
Removed constant and i18n